src/orders/components/OrderTransaction/components/TransactionEvents/components/EventStatus.tsx

```diff
@@ -16,30 +16,41 @@ export const EventStatus: React.FC<EventStatusProps> = ({ status }) => {
       return (
         <Pill
           color="warning"
+          size={"small"}
```

Comment:
If we don't that Prettier linting will fail (I'm not sure if it's done right now in dashboard's CI 😅)

```suggestion
          size="small"
```